splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkTrustManager.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.console.notifications.splunkintegration;
+
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+import javax.net.ssl.X509TrustManager;
+
+class SplunkTrustManager implements X509TrustManager {
+
+    public void checkClientTrusted(X509Certificate[] chain, String authType) throws CertificateException {
```

Comment:
What is the difference between server and client trusted methods? Can there only be one that would solve our case?

Reply:
client method:
`return if it can be validated and is trusted for client SSL authentication based on the authentication type.`

server method:
`return if it can be validated and is trusted for server SSL authentication based on the authentication type.`

We need the all three methods that are implemented, as they are inherited abstract methods from the interface